### ISSUE TITLE
Question more than anything...

### ISSUE THREAD

**Issue by dancwilliams** [IcyPalm/todoist-location-labels#1]
Hello,

Do you currently have this up and functioning?  I forked your repo and have it up in heroku and it _looks_ ok, but for some reason the address piece is not working for me.  I am able to login through Todoist and see my labels, but when I put in the address sometimes it autopopulates but still fails to pull lat/long.  Other times it just sits there.

I just wanted to make sure it is all working good on your end and I will continue to test on my end.  Thanks!

**Comment by dancwilliams** [comment]
Ugh...this was fun. It was an expired credit card on my Google account. Even though it was in the free tier the expired card wouldn’t let API requests process.

**Comment by IcyPalm** [comment]
Ah, yeah, I did experience that same issue. Spent WAY too many hours trying to fix it 🤦‍♂

**Comment by IcyPalm** [comment]
You have it up, running and working for you? ❓ 😃

**Comment by dancwilliams** [comment]
It is up and working for the most part. For some reason when I remove a label the location reminder is not being removed. I am looking into that today/tomorrow as I have time.

**Comment by dancwilliams** [comment]
Figured out my issue.  It is that the maximum radius Todoist will accept is 255, no matter what you send over that.  I had setup a location with a 500 meter radius and one with a 300 meters.  The API did not kick back an error, but when I pulled reminders from Todoist both were set to 255.  So, I added some helper text to the redius field and may revisit it later.  Also going to send a note to Todoist to add that maximum into their API docs.
